index.js

```diff
@@ -1 +1,72 @@
-// implement your API here
+const express = require('express');
+const cors = require('cors');
+const db = require('./data/db');
+const server = express();
```

Comment:
This is just a small optimization thing, but maybe you'd get some use out of it: since `express` here is just a reference to what you required on line 1, you can actually merge the two as `const server = require('express')()` and take the middleman out. This is the best way to learn what's happening though